Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "macaroon"
-version = "0.2.0"
+version = "0.3.0-dev.0"
 edition = "2018"
```

Comment:
It was mentioned that the edition is updated in the changelog but not actually done here. (Or is there a dependent PR missing?)

Reply:
Oh! Good catch. I was just updating the changelog from an earlier commit, but I now notice that commit is inconsistent (it mentions 2021 edition, but only sets 2018 edition). I will update this PR to remove the mention and fix the edition notes separately.

https://github.com/macaroon-rs/macaroon/commit/ba26cb8c473ad802430d646d8d664eaebc8d47ce